spec/std/dir_spec.cr

```diff
@@ -387,6 +405,10 @@ describe "Dir" do
     end
   end
 
+  pending_win32 ".current" do
+    Dir.current.should eq(`#{{{ flag?(:win32) ? "cd" : "pwd" }}}`.chomp)
```

Comment:
Why the interpolation?

Reply:
This is all correct, there's no other way to put a string into ``.

On the other hand, there's no plan to make this construct work like this on Windows, so it's not good to add it pending like this. You can use `cmd /c cd` maybe...

Reply:
Not true, see https://carc.in/#/r/8xpe

Reply:
Fine. The only safe way.

https://carc.in/#/r/8xpl
https://carc.in/#/r/8xpp

I **just** fixed a few of these. It'd be a change for worse.

https://github.com/crystal-lang/crystal/pull/9054/files#diff-42858a99f8c45d6bcda97f09679eea8dL284
https://github.com/crystal-lang/crystal/pull/9106/files#diff-a49a030cea42f05be9fa6ee72cb04404L10

Reply:
And the issue where I describe the latter fix: https://github.com/crystal-lang/crystal/issues/9089